_sass/_components/blog-post.scss

```diff
@@ -0,0 +1,197 @@
+// Blog post
+// ==========================
+
+.post-feature_image {
+  background-position: center;
+  background-size: cover;
+  height: 36rem;
+
+  @include media($medium-screen) {
+    height: 40rem;
+  }
+}
+
+.header_border,
+.single-post {
+  header[role=banner] {
+    border-bottom: 1px solid #cccccc; // See if we can use a WDS color
+  }
+}
+
+.single-post {
+  header[role=banner] {
+//    border-bottom: 1px solid #cccccc; // See if we can use a WDS color
+  }
+
+  article {
+    @include margin(0 null $section-margins null);
+
+    @include media($medium-screen + 200px) {
+      @include margin($section-margins auto);
+      max-width: 61rem;
+    }
+
+    .image-feature-small {
+      margin-top: 0;
+    }
+
+    img {
+      @include margin($section-margins null);
+    }
+
+    figure {
+      @include margin($section-margins null);
+      text-align: center; // center image if less than full width
+
+      img {
+        @include margin(0 null);
+      }
```

Comment:
It's nested in the page `.single-post` inside `article` - I think these are both necessary for this style.